src/pivot-table/hooks/use-grid-height.ts

```diff
@@ -24,5 +24,15 @@ export default function useGridHeight({ pageInfo, headersData, topDimensionData,
   const rowsCanFitInTableViewPort = Math.floor(tableRect.height / contentCellHeight);
   const showLastBottomBorder = pageInfo.rowsOnCurrentPage < rowsCanFitInTableViewPort;
 
-  return { containerHeight, topGridHeight, leftGridHeight, dataGridHeight, showLastBottomBorder };
+  const isEmptySpaceExistsBelowLastRow = topGridHeight + GRID_BORDER + dataGridHeight < tableRect.height;
```

Comment:
I would think this should be the same value as `showLastBottomBorder`? But `showLastBottomBorder` does appear to be bugged as it does not take `topGridHeight + GRID_BORDER` into account?

Reply:
fixed it with latest commit, 
but in terms of equality (showLastBottomBorder == isEmptySpaceExistsBelowLastRow) not quite sure if they are representing same value since `showLastBottomBorder` is computed value based on (header/content)rowHeight but `isEmptySpaceExistsBelowLastRow` is the explicit grid heights. so i think `isEmptySpaceExistsBelowLastRow` is much more precise (not Math.floor used for it) + in next PR, i have to consider horizontal scrollbar height into this calculation, so the purpose of them are different as well

https://github.com/qlik-oss/sn-pivot-table/assets/29652890/0babcf5a-3b36-4a7e-b378-58e76faec865

Reply:
`showLastBottomBorder` should be true only if all rows are visible to the user? Here is an example where it fails with that:


https://github.com/qlik-oss/sn-pivot-table/assets/16608020/3b59cc01-c1e9-48b2-bd5d-2238621501c0


I'm not really sure what the purpose of `isEmptySpaceExistsBelowLastRow` is, it it's not the same?

Reply:
I think `isEmptySpaceExistsBelowLastRow` is the only variable we should use. But perhaps reverse the logic and rename it to `areAllRowsVisible`? So `showLastBottomBorder` can be removed since it's broken